README.md

```diff
@@ -1,19 +1,27 @@
-![Swift](https://img.shields.io/badge/Swift-5.0-orange.svg)
-![platform: iOS, tvOS and macOS](https://img.shields.io/badge/platform-iOS%20%7C%20tvOS%20%7C%20macOS-lightgrey.svg)
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fjathu%2FUIImageColors%2Fbadge%3Ftype%3Dswift-versions)](https://swiftpackageindex.com/jathu/UIImageColors)
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fjathu%2FUIImageColors%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/jathu/UIImageColors)
+[![Xcode Build](https://github.com/jathu/UIImageColors/actions/workflows/xcodebuild.yml/badge.svg)](https://github.com/jathu/UIImageColors/actions/workflows/xcodebuild.yml)
 
 # UIImageColors
 
 iTunes style color fetcher for `UIImage` and `NSImage`. It fetches the most dominant and prominent colors.
 
-![preview](preview.png)
+<p float="left">
+  <img src="/preview-iphone.png" width="30%"/>
+  <img src="/preview-macos.png" width="65%"/> 
+</p>
```

Comment:
It's unnecessary to have two previews with a different device border. Let's just include a single cropped preview like before

Reply:
Do you not like how this looks? I think that makes much more clear that it's available on both platforms.

<img width="793" alt="Screenshot 2021-10-31 at 00 23 46" src="https://user-images.githubusercontent.com/42500484/139560009-251a9fba-5188-4a7a-bde3-8a9877c8ede2.png">